extensions/amp-bind/0.1/test/test-bind-validator.js

```diff
@@ -146,16 +146,22 @@ describe('BindValidator', () => {
       expect(val.isResultValid('A', 'href',
           /* eslint no-script-url: 0 */ 'javascript:alert(1)')).to.be.false;
       expect(val.isResultValid('A', 'href',
+          /* eslint no-script-url: 0 */ 'javascript:alert(1)\n')).to.be.false;
```

Comment:
let's change this to `javascript:alert(1)\n;` (note the `;`) to cover the unlikely case that regex is switched back to old one and a `trim` is added before validating.